frontend/src/components/order/order.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  justify-content: space-around;
+  border-color: black;
+  border-width: 2px;
+  border-radius: 10px;
+  border-style: solid;
+  font-size: 1.5rem;
+}
+
+.green {
```

Comment:
I don't see this or red used anywhere, but if we are using them, could we give them a name that specifies what we are making red or green or pair them with another style object?

Reply:
I'll get rid of them for now